### PR TITLE
fix(doc): Update LTS patch release notes

### PR DESCRIPTION
**Pull request by abhipsaMisra**
Since the LTS release is getting delayed from the previous estimate we need to update the dates

### CODE REVIEW

**Comment by abhipsaMisra** [comment]
Waiting to merge this in after we get the sign-off from the edge team.

**Review by timtay-microsoft — APPROVED** [review]

**Review by barustum — APPROVED** [review]

**Review by jamdavi — APPROVED** [review]

**Pull request merged**
